club.haroohie.SerialLoops.yaml

```diff
@@ -0,0 +1,83 @@
+id: club.haroohie.SerialLoops
+runtime: org.kde.Platform
+runtime-version: '6.9'
+base: net.kuribo64.melonDS
+base-version: stable
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+  - org.freedesktop.Sdk.Extension.llvm20
+build-options:
+  prepend-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+  prepend-pkg-config-path: "/usr/lib/sdk/dotnet8/lib/pkgconfig"
+
+command: SerialLoops
+
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --filesystem=~/SerialLoops:create
+  - --env=DOTNET_ROOT=/app/lib/dotnet
+  - --env=SL_LLVM_PATH=/app/lib/llvm20
+  - --env=SL_NINJA_PATH=/app/bin/ninja
+  - --env=SL_BUNDLED_EMULATOR=/app/bin/melonDS
+  - --env=SL_STORE_SYS_CONFIG=false
+  - --env=SL_USE_UPDATER=false
+
+modules:
+  - name: dotnet
+    buildsystem: simple
+    build-commands:
+    - /usr/lib/sdk/dotnet8/bin/install.sh
+
+  - name: llvm
+    buildsystem: simple
+    build-commands:
+    - mkdir -p ${FLATPAK_DEST}/lib/llvm20/bin/
+    - mkdir -p ${FLATPAK_DEST}/lib/llvm20/lib/
+    - cp /usr/lib/sdk/llvm20/bin/clang ${FLATPAK_DEST}/lib/llvm20/bin
+    - cp /usr/lib/sdk/llvm20/bin/ld.lld ${FLATPAK_DEST}/lib/llvm20/bin
+    - cp /usr/lib/sdk/llvm20/bin/llvm-objcopy ${FLATPAK_DEST}/lib/llvm20/bin
+    - cp /usr/lib/sdk/llvm20/bin/llvm-objdump ${FLATPAK_DEST}/lib/llvm20/bin
+    - cp -r /usr/lib/sdk/llvm20/lib/* ${FLATPAK_DEST}/lib/llvm20/lib/
+
+  - name: Ninja
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/ninja-build/ninja.git
+        tag: v1.13.1
+    build-commands:
+      - python3 ./configure.py --bootstrap
+      - mv ninja ${FLATPAK_DEST}/bin/ninja
+
+    # Our SDL library expects libSDL to be in a specific spot, so we copy it here
+  - name: SDL
+    buildsystem: simple
+    build-commands:
+      - cp /usr/lib/x86_64-linux-gnu/libSDL* ${FLATPAK_DEST}/lib/
```

Comment:
this won't work on aarch64

Reply:
there is no reason to do it anyway. It's a broken build system at the core.